spec.html

```diff
@@ -32512,11 +32532,12 @@ <h1>Static Semantics: ParsePattern ( _patternText_, _u_ )</h1>
           <p>The abstract operation ParsePattern takes arguments _patternText_ (a sequence of Unicode code points) and _u_ (a Boolean). It performs the following steps when called:</p>
           <emu-alg>
             1. If _u_ is *true*, then
-              1. Parse _patternText_ using the grammars in <emu-xref href="#sec-patterns"></emu-xref>. The goal symbol for the parse is |Pattern[+U, +N]|.
+              1. Let _parseResult_ be ParseText(_patternText_, |Pattern[+U, +N]|).
```

Comment:
You can just return directly here. Similarly below.

Reply:
I tried out the direct returns and found I didn't like the result.